app/views/rooms/show.html.erb

```diff
@@ -101,7 +151,9 @@ $(function(){
         $("#review_content").val(" ")
         var $ol = $("div.reviews ol")
         $ol.append(response);
+        bindDeleteReviewEventListeners();
```

Comment:
is there a way to just bind a delete event to brand new element instead of rebinding all of the delete buttons from the initial get request for reviews?